base_agent/documents/logical_form_specification/filters.md

```diff
@@ -55,8 +55,8 @@ This is used to mean the number of steps (in "has_measure" units, default is "bl
 {"linear_extent" : {
             "relative_direction": "LEFT" / "RIGHT"/ "UP" / "DOWN"/ "FRONT" 
                                   / "BACK"/ "AWAY" / "INSIDE" / "OUTSIDE", 
-            "frame": "SPEAKER" / "AGENT" / "ABSOLUTE" / {"player_span": span},
-            "has_measure" : span,
+            "frame": {"fixed_value": "SPEAKER" / "AGENT" / "ABSOLUTE", "player_span": span},
```

Comment:
slightly not clear from this, bc the comma: it should either be fixed_value xor player_span, but we have been using the comma to denote possibility of non-exclusive or

Reply:
Right yeah: so we'd either have "fixed_value" which can have one of the values (the list of which is separated by `/`) or we have "player_span". 
Would something like this make it clearer:
```
"frame": {"fixed_value": "SPEAKER" / "AGENT" / "ABSOLUTE"}  /  {"player_span": span}
```